sdk/communication/communication-chat/swagger/README.md

```diff
@@ -10,13 +10,13 @@ title: ChatApiClient
 description: Chat Client
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-tag: package-2023-11-15-preview
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/de868d17c7e25379f826fa43963d1ce6015cded3/specification/communication/data-plane/Chat/readme.md
+tag: package-2024-03-01-preview
+require: https://github.com/Azure/azure-rest-api-specs/blob/f1a7edf32c54d448793023fdbfc63f24c32b5417/specification/communication/data-plane/Chat/readme.md
 model-date-time-as-string: false
 optional-response-headers: true
 add-credentials: false
 disable-async-iterators: true
-package-version: 1.5.0-beta.1
+package-version: 1.5.0-beta.2
```

Comment:
Please run a `rushx format` for the code and please make sure the recording tests are passing for the playback

Reply:
Once https://github.com/Azure/azure-rest-api-specs/pull/28565 is merged, I will update the tests.

Reply:
Note the format failure is unrelated to chat:
> Invoke "rushx format" inside sdk/communication/communication-toll-free-verification to fix formatting